src/Graph.h

```diff
@@ -116,26 +116,62 @@ class Graph
     unsigned int getSize();
 
     /**
+     * Dijkstra method
      * findPathLengths() will get all the path lengths with a certain node from the center.
      * @param start the start node
      * @param metric an integer specifying the data value that we want to create the minimum
-     * paths with. Metric is 0 for fare (cost), 1 for miles (length), and 2 for time.
+     * paths with. Metric is 0 for fare (cost), 1 for time, and 2 for miles (length)
      * @return a vector of pairs of nodes and the length from the start node.
      * The length from the start node to itself is 0.
      */
     std::vector<std::pair<Node*, double>> findPathLengths(Node* start, int metric);
 
     /**
+     * Dijkstra method
      * findShortestPath() uses findPathLengths() and finds the length between 2 points given a certain metric.
      * @param start the start node
      * @param end the end node
      * @param metric an integer specifying the data value that we want to create the minimum
-     * paths with. Metric is 0 for fare (cost), 1 for miles (length), and 2 for time.
-     * @return a vector of pairs of nodes and the length from the start node.
+     * paths with. Metric is 0 for fare (cost), 1 for time, and 2 for miles (length)
+     * @return the length from the start node to end node
      * The length from the start node to itself is 0.
      */
     double findShortestPath(Node* start, Node* end, int metric);
 
+    /**
+     * Betweenness Centrality method (more methods below as well)
+     * findShortestPath_PassByNode() uses findShortestPath() and finds the number of 
+     * shortest paths between the nodes start and end that pass-through the node given a certain metric
+     * @param start 
+     * @param node 
+     * @param end 
+     * @param metric Metric is 0 for fare (cost), 1 for time, and 2 for miles (length)
+     * @return double 
+     */
+    double findShortestPath_PassByNode(Node* start, Node* node, Node* end, int metric);
+
+    /**
+     * find all combinations or all pairs of nodes in the graph excluding the Node node
+     * sets the private variable called node_pairs
+     * @param std::vector<int> n
+     */
+    void combinations2(std::vector<Node*> n);
+
+    /**
+     * populateRelations() uses node_pairs to populate 
```

Comment:
i like how you stated the funciton name.  Although, this is gibberish to me.  Make sure the comments are short and sweet yet understandble to the purpose of what it does.  You did a great job in the other ones above.